demo/src/screens/nativeComponentScreens/keyboardInput/KeyboardInputViewScreen.js

```diff
@@ -0,0 +1,145 @@
+import React, {PureComponent} from 'react';
+import {ScrollView, StyleSheet, TextInput} from 'react-native';
+import {Keyboard, Text, View, Colors, Spacings, Constants, Typography, Button} from 'react-native-ui-lib';
+const KeyboardAccessoryView = Keyboard.KeyboardAccessoryView;
+const KeyboardUtils = Keyboard.KeyboardUtils;
+
+import './demoKeyboards';
+const KeyboardRegistry = Keyboard.KeyboardRegistry;
+
+const TrackInteractive = true;
+
+export default class KeyboardInputViewScreen extends PureComponent {
+  state = {
+    customKeyboard: {
+      component: undefined,
+      initialProps: undefined
+    },
+    receivedKeyboardData: undefined
+  };
+
+  onKeyboardItemSelected = (keyboardId, params) => {
+    const receivedKeyboardData = `onItemSelected from "${keyboardId}"\nreceived params: ${JSON.stringify(params)}`;
+    this.setState({receivedKeyboardData});
+  };
+
+  onKeyboardResigned = () => {
+    this.resetKeyboardView();
+  };
+
+  getToolbarButtons() {
+    const keyboards = KeyboardRegistry.getAllKeyboards();
+    const buttons = [];
+    for (let index = 0; index < keyboards.length; ++index) {
+      const string = `Show KB ${index + 1}`;
+      const title = `Title ${index + 1} (passed prop)`;
+      buttons.push({
+        text: string,
+        testID: string,
+        onPress: () => this.showKeyboardView(keyboards[index].id, title)
+      });
+    }
+
+    buttons.push({
+      text: 'reset',
+      testID: 'reset',
+      onPress: () => this.resetKeyboardView()
+    });
+
+    return buttons;
+  }
+
+  resetKeyboardView = () => {
+    this.setState({customKeyboard: {}});
+  };
+
+  showKeyboardView(component, title) {
+    this.setState({
+      customKeyboard: {
+        component,
+        initialProps: {title}
+      }
+    });
+  }
+
+  onHeightChanged = keyboardAccessoryViewHeight => {
```

Comment:
This is not really used, but was there on the original screen (without usage) - should we leave it as a placeholder for users?